packages/salesforce-adapter/test/change_validators/standard_field_or_object_addition.test.ts

```diff
@@ -0,0 +1,61 @@
+/*
+*                      Copyright 2023 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+import {
+  toChange,
+} from '@salto-io/adapter-api'
+import { mockTypes } from '../mock_elements'
+import changeValidator from '../../src/change_validators/standard_field_or_object_addition'
+
+describe('Adding or removing standard objects and fields', () => {
+  describe('Addition or removal of standard object', () => {
+    it('should have error for standard object creation', async () => {
+      const standardObjectAdditionChange = toChange({ after: mockTypes.Account })
+      const changeErrors = await changeValidator([standardObjectAdditionChange])
+      expect(changeErrors).toHaveLength(1)
+      const [changeError] = changeErrors
+      expect(changeError.elemID).toEqual(mockTypes.Account.elemID)
+      expect(changeError.severity).toEqual('Error')
```

Comment:
```suggestion
      expect(changeErrors).toEqual([
        expect.objectContaining({
          elemID: mockTypes.Account.elemID
          severity: 'Error',
        }),
      ])
```
Nit: Consider the following instead :)